sqlmesh/dbt/model.py

```diff
@@ -467,7 +468,7 @@ def to_sqlmesh(
                 partitioned_by.append(self._big_query_partition_by_expr(context))
             optional_kwargs["partitioned_by"] = partitioned_by
 
-        if self.cluster_by:
+        if self.cluster_by and not isinstance(kind, ViewKind):
```

Comment:
Don't we want to push this further down into sqlmesh's model validator? I thought it wasn't possible to cluster normal views in general.

Reply:
I see, that makes sense to me. Do we already throw an error today for this? (Just curious, fine if we don't for now.)